src/components/tutorials-sidebar/index.tsx

```diff
@@ -30,6 +35,66 @@ function TutorialsSidebar({
   )
 }
 
+function CollectionViewSidebarContent({
+  sections,
+}: {
+  sections: CollectionCategorySidebarSection[]
+}) {
+  return (
+    <>
+      {sections?.map((section: CollectionCategorySidebarSection) => {
+        return (
+          <>
+            <HorizontalRule />
+            <SectionTitle text={section.title} />
+            <SectionList items={section.items} />
+          </>
+        )
+      })}
+      <HorizontalRule />
+      <SectionTitle text="Resources" />
```

Comment:
💭 Seems like there's small opportunity to create a `Resources` component to share between these two views.